pkg/starter/scripts.go

```diff
@@ -49,7 +50,7 @@ func (s *Starter) getSetupRawScript(ctx context.Context, target datastore.Target
 	if target.RunnerUser.Valid {
 		runnerUser = target.RunnerUser.String
 	}
-	runnerVersion, runnerTemporaryMode, err := datastore.GetRunnerTemporaryMode(target.RunnerVersion)
+	runnerVersion, runnerTemporaryMode, err := runner.GetRunnerTemporaryMode(s.runnerVersion)
```

Comment:
runner_version in the bash script require `v2.xxx.yy` (format).
But `s.runnerVersion` is `latest` by default.

Please change to use the latest version in execute timing.

Reply:
Added checking for latest version.